### PR TITLE
Fix npm sub dependencies, Fix web3 750s error

### PR DESCRIPTION
**Pull request by pmprete**
Fix npm sub dependencies warned by Dependabot and npm audit
Handle web3 error when 750 seconds  have pased and transaction hasn't been mined.
Take the bigger block number between the config file and lastBlock procesed

### CODE REVIEW

**Review by jonathansmirnoff — APPROVED** [review]
LGTM

File: bridge/package-lock.json [hunk 1]
```diff
@@ -525,527 +568,768 @@
...
+            "web3-core": "1.2.1",
+            "web3-core-method": "1.2.1",
+            "web3-utils": "1.2.1"
           }
         },
         "web3-providers-http": {
-          "version": "1.2.2",
-          "resolved": "https://registry.npmjs.org/web3-providers-http/-/web3-providers-http-1.2.2.tgz",
-          "integrity": "sha512-BNZ7Hguy3eBszsarH5gqr9SIZNvqk9eKwqwmGH1LQS1FL3NdoOn7tgPPdddrXec4fL94CwgNk4rCU+OjjZRNDg==",
+          "version": "1.2.1",
```
Comment (bcodesido) [inline bridge/package-lock.json:1168]: downgrade?

File: bridge/package-lock.json [hunk 2]
```diff
@@ -525,527 +568,768 @@
...
-            "web3-core-helpers": "1.2.2",
+            "web3-core-helpers": "1.2.1",
             "xhr2-cookies": "1.1.0"
           }
         },
         "web3-providers-ipc": {
-          "version": "1.2.2",
-          "resolved": "https://registry.npmjs.org/web3-providers-ipc/-/web3-providers-ipc-1.2.2.tgz",
-          "integrity": "sha512-t97w3zi5Kn/LEWGA6D9qxoO0LBOG+lK2FjlEdCwDQatffB/+vYrzZ/CLYVQSoyFZAlsDoBasVoYSWZK1n39aHA==",
+          "version": "1.2.1",
```
Comment (bcodesido) [inline bridge/package-lock.json:1177]: downgrade?

File: bridge/package-lock.json [hunk 3]
```diff
@@ -525,527 +568,768 @@
...
             "underscore": "1.9.1",
-            "web3-core-helpers": "1.2.2"
+            "web3-core-helpers": "1.2.1"
           }
         },
         "web3-providers-ws": {
-          "version": "1.2.2",
-          "resolved": "https://registry.npmjs.org/web3-providers-ws/-/web3-providers-ws-1.2.2.tgz",
-          "integrity": "sha512-Wb1mrWTGMTXOpJkL0yGvL/WYLt8fUIXx8k/l52QB2IiKzvyd42dTWn4+j8IKXGSYYzOm7NMqv6nhA5VDk12VfA==",
+          "version": "1.2.1",
```
Comment (bcodesido) [inline bridge/package-lock.json:1187]: downgrade?

File: bridge/package-lock.json [hunk 4]
```diff
@@ -525,527 +568,768 @@
...
-            "web3-core-helpers": "1.2.2",
+            "web3-core-helpers": "1.2.1",
             "websocket": "github:web3-js/WebSocket-Node#polyfill/globalThis"
           }
         },
         "web3-shh": {
-          "version": "1.2.2",
-          "resolved": "https://registry.npmjs.org/web3-shh/-/web3-shh-1.2.2.tgz",
-          "integrity": "sha512-og258NPhlBn8yYrDWjoWBBb6zo1OlBgoWGT+LL5/LPqRbjPe09hlOYHgscAAr9zZGtohTOty7RrxYw6Z6oDWCg==",
+          "version": "1.2.1",
```
Comment (bcodesido) [inline bridge/package-lock.json:1197]: downgrade?

File: bridge/package-lock.json [hunk 5]
```diff
@@ -525,527 +568,768 @@
...
+            "web3-core-method": "1.2.1",
+            "web3-core-subscriptions": "1.2.1",
+            "web3-net": "1.2.1"
           }
         },
         "web3-utils": {
-          "version": "1.2.2",
-          "resolved": "https://registry.npmjs.org/web3-utils/-/web3-utils-1.2.2.tgz",
-          "integrity": "sha512-joF+s3243TY5cL7Z7y4h1JsJpUCf/kmFmj+eJar7Y2yNIGVcW961VyrAms75tjUysSuHaUQ3eQXjBEUJueT52A==",
+          "version": "1.2.1",
```
Comment (bcodesido) [inline bridge/package-lock.json:1208]: downgrade?

File: bridge/package-lock.json [hunk 6]
```diff
@@ -525,527 +568,768 @@
...
-          "requires": {
-            "bn.js": "^4.4.0",
-            "brorand": "^1.0.1",
-            "hash.js": "^1.0.0",
-            "hmac-drbg": "^1.0.0",
-            "inherits": "^2.0.1",
-            "minimalistic-assert": "^1.0.0",
-            "minimalistic-crypto-utils": "^1.0.0"
-          }
+          "version": "10.17.29",
```
Comment (bcodesido) [inline bridge/package-lock.json:1330]: downgrade?

File: bridge/package-lock.json [hunk 7]
```diff
@@ -1073,248 +1357,144 @@
...
-        "js-sha3": {
-          "version": "0.5.7",
-          "resolved": "https://registry.npmjs.org/js-sha3/-/js-sha3-0.5.7.tgz",
-          "integrity": "sha1-DU/9gALVMzqrr0oj7tL2N0yfKOc="
-        },
         "scrypt-js": {
-          "version": "2.0.4",
-          "resolved": "https://registry.npmjs.org/scrypt-js/-/scrypt-js-2.0.4.tgz",
-          "integrity": "sha512-4KsaGcPnuhtCZQCxFxN3GVYIhKFPTdLd8PLC552XwbMndtD0cjRFAhDuuydXQ0h08ZfPgzqe6EKHozpuH74iDw=="
+          "version": "2.0.3",
```
Comment (bcodesido) [inline bridge/package-lock.json:1361]: downgrade?

File: bridge/package-lock.json [hunk 8]
```diff
@@ -1073,248 +1357,144 @@
...
         "uuid": {
           "version": "2.0.1",
           "resolved": "https://registry.npmjs.org/uuid/-/uuid-2.0.1.tgz",
           "integrity": "sha1-wqMN7bPlNdcsz4LjQ5QaULqFM6w="
         },
         "web3": {
-          "version": "1.2.2",
-          "resolved": "https://registry.npmjs.org/web3/-/web3-1.2.2.tgz",
-          "integrity": "sha512-/ChbmB6qZpfGx6eNpczt5YSUBHEA5V2+iUCbn85EVb3Zv6FVxrOo5Tv7Lw0gE2tW7EEjASbCyp3mZeiZaCCngg==",
+          "version": "1.2.1",
```
Comment (bcodesido) [inline bridge/package-lock.json:1381]: downgrade?

**Review by bcodesido — CHANGES_REQUESTED** [review]
Why are you downgrading dependencies?

File: bridge/package-lock.json [hunk 9]
```diff
@@ -23,413 +23,469 @@
...
         "@babel/helper-validator-identifier": "^7.10.4",
         "lodash": "^4.17.19",
         "to-fast-properties": "^2.0.0"
       }
     },
     "@ethersproject/abi": {
-      "version": "5.0.1",
-      "resolved": "https://registry.npmjs.org/@ethersproject/abi/-/abi-5.0.1.tgz",
-      "integrity": "sha512-9fqSa3jEYV4nN8tijW+jz4UnT/Ma9/b8y4+nHlsvuWqr32E2kYsT9SCIVpk/51iM6NOud7xsA6UxCox9zBeHKg==",
+      "version": "5.0.0-beta.153",
```
[the remark this answers is not in the document]
Comment (pmprete) [reply bridge/package-lock.json:55]: ask web3 why they downgraded their sub dependency version
"web3-eth-abi": {
          "version": "1.2.11",
          "resolved": "https://registry.npmjs.org/web3-eth-abi/-/web3-eth-abi-1.2.11.tgz",
          "integrity": "sha512-PkRYc0+MjuLSgg03QVWqWlQivJqRwKItKtEpRUaxUAeLE7i/uU39gmzm2keHGcQXo3POXAbOnMqkDvOep89Crg==",
          "requires": {
            "@ethersproject/abi": "5.0.0-beta.153",
            "underscore": "1.9.1",
            "web3-utils": "1.2.11"
          }
        },

File: bridge/package-lock.json (hunk 1 above)
Comment (pmprete) [reply bridge/package-lock.json:1168]: automatically generated please see package.lock to see that they wheren't done on our side, can't be responsible by the decision on the packages chosen by their creators

**Review by jonathansmirnoff — APPROVED** [review]
LGTM

**Review by bcodesido — APPROVED** [review]

**Pull request merged**